docs/docs/classic-ml/search/search-models/index.mdx

```diff
@@ -0,0 +1,525 @@
+# Find recently created models (timestamp in milliseconds)
+import time
+
+last_week = int((time.time() - 7 * 24 * 60 * 60) * 1000)
+
+recent_models = mlflow.search_logged_models(
+    experiment_ids=["1"], filter_string=f"creation_time > {last_week}"
+)
+```
+
+### 5 - Dataset-Specific Metric Filtering
+
+One powerful feature of `search_logged_models` is the ability to filter metrics based on specific datasets:
+
+```python
+# Find models with high accuracy on test dataset
+test_accurate_models = mlflow.search_logged_models(
+    experiment_ids=["1"],
+    filter_string="metrics.accuracy > 0.9",
+    datasets=[{"dataset_name": "test_dataset", "dataset_digest": "abc123"}],  # Optional
+)
+
+# Multiple dataset conditions
+multi_dataset_models = mlflow.search_logged_models(
+    experiment_ids=["1"],
+    filter_string="metrics.accuracy > 0.85",
+    datasets=[{"dataset_name": "test_dataset"}, {"dataset_name": "validation_dataset"}],
+)
+```
+
+### 6 - Complex Queries
+
+Combine multiple conditions for sophisticated model discovery:
+
+```python
+# Production-ready RandomForest models with high performance
+production_ready = mlflow.search_logged_models(
+    experiment_ids=["1"],
+    filter_string="""
+        params.model_type = 'RandomForest'
+        AND metrics.accuracy > 0.9
+        AND metrics.precision > 0.88
+    """,
+)
+```
+
+## Programmatic Search with Python
+
+The Python API provides powerful capabilities for searching logged models programmatically.
+
+### Using the Fluent API
+
+<APILink fn="mlflow.search_logged_models" /> provides a convenient interface for model search:
+
+```python
+import mlflow
+
+# Basic search with pandas output (default)
+models_df = mlflow.search_logged_models(
+    experiment_ids=["1"], filter_string="metrics.accuracy > 0.9"
+)
+
+# Check available columns
+print("Available columns:", models_df.columns.tolist())
+print("\nModel information:")
+print(models_df[["name", "source_run_id"]])
+
+# Get results as a list instead of DataFrame
+models_list = mlflow.search_logged_models(
+    experiment_ids=["1"], filter_string="metrics.accuracy > 0.9", output_format="list"
+)
+
+for model in models_list:
+    print(f"Model: {model.name}, Run ID: {model.source_run_id}")
+```
+
+### Using the Client API
+
+<APILink fn="mlflow.client.MlflowClient.search_logged_models" /> offers more control with pagination support:
+
+```python
+from mlflow import MlflowClient
+
+client = MlflowClient()
+
+# Search with pagination
+page_token = None
+all_models = []
+
+while True:
+    result = client.search_logged_models(
+        experiment_ids=["1"],
+        filter_string="metrics.accuracy > 0.85",
+        max_results=10,
+        page_token=page_token,
+    )
+
+    all_models.extend(result.to_list())
+
+    if not result.token:
+        break
+    page_token = result.token
+
+print(f"Found {len(all_models)} models")
+```
+
+### Advanced Ordering
+
+Control the order of search results using the `order_by` parameter:
```

Comment:
Let's mention that `order_by` must be a list of dictionaries with `field_name` and optionally `ascending` keys.

Reply:
Added a tip section :)